e2e-runner/lib/darwin/runner.sh

```
@@ -455,13 +466,17 @@ if (( extTests == 1 )); then
     cd "$workingDir/$extRepo"
     echo "Installing dependencies of $extRepo"
     pnpm install --frozen-lockfile 2>&1 | tee -a $testsOutputLog
+    restore_deferred_secrets
     echo "Running the e2e playwright tests using target: $npmTarget"
     pnpm $npmTarget 2>&1 | tee -a $testsOutputLog
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)